route/build/src/historyFile.f90

```diff
@@ -309,6 +321,15 @@ SUBROUTINE createNC_rch_hru(this, nRch_in, nHRU_in, ierr, message)
         if(ierr/=0)then; message=trim(message)//trim(cmessage); return; endif
       end do
 
+      call put_attr(this%pioFileDesc, ncd_global, 'mizuRoute-version', version ,ierr, cmessage)
+      if(ierr/=0)then; message=trim(message)//trim(cmessage); return; endif
+
+      call put_attr(this%pioFileDesc, ncd_global, 'gitBranch', gitBranch ,ierr, cmessage)
```

Comment:
For the coupled case, we don't have a way to fill gitBranch or gitHash.

We could maybe add this by adding it to the mizuRoute control file? Or maybe it's OK if it's empty for the CESM coupled case?

Reply:
it is ok if they are not available.